content/en/profiler/enabling/ddprof.md

```diff
@@ -26,7 +26,7 @@ Profiles sent from `ddprof` show up under the _native_ runtime in the Datadog we
 ## Requirements
 
 Supported operating systems
-: Linux kernel v4.17+, glibc 2.18+ or musl libc
+: Linux (glibc or musl)
```

Comment:
We have tested with older kernel versions, though I'm not exactly sure what should be the minimum if we were to give one. 
Also it can be confusing for users to see a minimum, especially when it is unlikely that they would hit it.